src/lib/ydocker/pull_image_dialog.rb

```diff
@@ -0,0 +1,111 @@
+# Copyright (c) 2017 SUSE LLC.
+#  All Rights Reserved.
+
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of version 2 or 3 of the GNU General
+# Public License as published by the Free Software Foundation.
+
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.   See the
+#  GNU General Public License for more details.
+
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, contact SUSE LLC.
+
+#  To contact Novell about this file by physical or electronic mail,
+#  you may find current contact information at www.suse.com
+
+
+require "docker"
+require "yast"
+
+module YDocker
+  class PullImageDialog
+    include Yast::UIShortcuts
+    include Yast::I18n
+    extend Yast::I18n
+
+    def initialize()
```

Comment:
parens not needed. Maybe time to add rubocop even to yast2-docker? we already use it in some modules and also have shared config with yast unification - see e.g. https://github.com/yast/yast-bootloader/blob/master/.rubocop.yml#L3

Reply:
> parens not needed. Maybe time to add rubocop even to yast2-docker? we already use it in some modules and also have shared config with yast unification - see e.g. https://github.com/yast/yast-bootloader/blob/master/.rubocop.yml#L3

Deleted. I will try to do a pull request to include rubocop.